### DISCUSSION TITLE
REM Configs

### DISCUSSION THREAD

**Discussion by gw8674** [tagyoureit/nodejs-poolController#868] [Q&A] [answered]
I finally got around to pulling latest REM commits (it's been awhile) and just doing some REM housekeeping.
I think I knew the answers to the following questions three years ago when we initially set up REM but I seemed to have forgotten the recommended settings.

1. In the I2C bus device settings pages, is it recommended to check the "Lock Protocol" checkbox on the I2C devices?
2. What is the recommended "Read Interval"  settings for the following devices?
EZO-ORP, EZO-pH, EZO-EC, EZO-RTD's, EZO-PRS & EZO-HUM?

Thanks much!!!

**Comment by rstrouse** [comment, marked as answer]
Mine are all set to around 5 seconds.  Lock protocol doesn’t really matter.

**Reply by gw8674** [reply]
Cool... Thanks much!
Just curious.. What does the lock protocol checkbox do?

**Reply by rstrouse** [reply]
There is a complete description of it in the EZO docs but it is unclear as to what benefit it hase.

**Reply by lymanepp** [reply]
From https://envirodiy.github.io/ModularSensors/group__atlas__group.html

"I recommend locking the protocol (plock) to I2C so the sensors do not accidentally switch back to UART mode".
